DotNetCasClient/Validation/CasSaml11Response.cs

```diff
@@ -98,7 +98,7 @@ private void ProcessValidAssertion()
         {
             protoLogger.Debug("Unmarshalling SAML response");
             XmlDocument document = new XmlDocument();
-            document.Load(new StringReader(_CasResponse));
+            document.LoadXml(_CasResponse);

```

Comment:
Technically code before and after is the same, you can test that by comparing source code (in reference sources or via DotPeek / Reflector)

The one was before change https://github.com/Microsoft/referencesource/blob/master/System.Xml/System/Xml/Dom/XmlDocument.cs#L1087

```
public virtual void Load( TextReader txtReader ) {
    XmlTextReader reader = SetupReader( new XmlTextReader( txtReader, NameTable ) );
    try {
        Load( reader );
    }
    finally {
        reader.Impl.Close( false );
    }
}
```

The one I use after change https://github.com/Microsoft/referencesource/blob/master/System.Xml/System/Xml/Dom/XmlDocument.cs#L1120

```
public virtual void LoadXml( string xml ) {
    XmlTextReader reader = SetupReader( new XmlTextReader( new StringReader( xml ), NameTable ));
    try {
        Load( reader );
    }
    finally {
        reader.Close();
    }
}
```

Differences between methods (i.e. before fix and after fix):
* `new StringReader( xml )` instead of `txtReader`
* not closing input reader if that was passed from outside: `reader.Impl.Close( false )` instead of `reader.Close()` - as a result of change it is closed no matter what, and that's the fix

So as you can see that's too just the fix of not disposing IDisposable.

If I haven't convince you please feel free to discard / to edit as you like, edits from maintainers allowed